app/serializers/atmosphere/action_log_serializer.rb

```diff
@@ -0,0 +1,10 @@
+#
+# Action log serializer.
+#
+module Atmosphere
+  class ActionLogSerializer < ActiveModel::Serializer
+
+    attributes :id, :log_level, :message, :created_at
```

Comment:
Are you sure? Action has many ActionLogs, thus inside ActionLog there is `action_id` (see line 22 in schema.db)